projects/js-packages/publicize-components/src/components/review-prompt/index.jsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * Panel that requests a review of the Jetpack Social Plugin
+ * Shows in the post publish panel of the editor
+ */
+
+import { Button, ThemeProvider } from '@automattic/jetpack-components';
+import { __ } from '@wordpress/i18n';
+import { useCallback } from 'react';
+import useAnalyticsTracks from '../../hooks/use-anaytics-tracks';
+import styles from './styles.module.scss';
+
+const ReviewPrompt = props => {
+	const { href, onClose } = props;
```

Comment:
```suggestion
const ReviewPrompt = ( { href, onClose } ) => {
```

Just a nitpick, but if we want to deconstruct all props anyway, why not to do it in param declaration?

Reply:
Nice suggestion, thanks 👍